test/views/CalculationViewSpec.scala

```diff
@@ -107,6 +108,10 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
           doc.select(Selector.breadcrumbWithLink(2)).attr("href") shouldBe appConfig.vatOverviewUrl
           doc.select(Selector.breadcrumbWithLink(3)).attr("href") shouldBe controllers.routes.IndexController.onPageLoad.url
         }
+
+        "not display TPP content" in {
+          doc.select("#ttp-inset-text").isEmpty shouldBe true
```

Comment:
If you add the class I mentioned, you can then do 
```suggestion
          doc.select(".whatever-class").isEmpty shouldBe true
```